src/natcap/invest/routedem.py

```diff
@@ -94,15 +92,168 @@
         "filled.tif": spec_utils.FILLED_DEM,
         "flow_accumulation.tif": spec_utils.FLOW_ACCUMULATION,
         "flow_direction.tif": spec_utils.FLOW_DIRECTION,
         "slope.tif": spec_utils.SLOPE,
         "stream_mask.tif": spec_utils.STREAM,
-        "_taskgraph_working_dir": spec_utils.TASKGRAPH_DIR
+        "strahler_stream_order.gpkg": {
+            "about": (
+                "A vector of line segments indicating the Strahler stream "
+                "order and other properties of each stream segment."),
+            "geometries": spec_utils.LINESTRING,
+            "fields": {
+                "order": {
+                    "about": "The Strahler stream order.",
+                    "type": "number",
+                    "units": u.none,
+                },
+                "river_id": {
+                    "about": (
+                        "A unique identifier used by all stream segments that "
+                        "connect to the same outlet."),
+                    "type": "number",
+                    "units": u.none,
+                },
+                "drop_distance": {
+                    "about": (
+                        "The drop distance in DEM elevation units from the "
+                        "upstream to downstream component of this stream "
+                        "segment."),
+                    "type": "number",
+                    "units": u.none,
+                },
+                "outlet": {
+                    "about": (
+                        "1 if this segment is an outlet, 0 if it is not."),
+                    "type": "number",
+                    "units": u.none,
+                },
+                "us_fa": {
+                    "about": (
+                        "The flow accumulation value at the upstream end of "
+                        "the stream segment."),
+                    "type": "number",
+                    "units": u.count,
+                },
+                "ds_fa": {
+                    "about": (
+                        "The flow accumulation value at the downstream end of "
+                        "the stream segment."),
+                    "type": "number",
+                    "units": u.pixels,  # TODO: does this work?
+                },
+                "thresh_fa": {
+                    "about": (
+                        "The final threshold flow accumulation value used to "
+                        "determine the river segments."),
+                    "type": "number",
+                    "units": u.pixels,  # TODO: does this work?
+                },
+                "upstream_d8_dir": {
+                    "about": (
+                        "The direction of flow immediately upstream."),
+                    "type": "number",
+                    "units": u.none,
+                },
+                "ds_x": {
+                    "about": "The raster X coordinate for the outlet.",
```

Comment:
Is this specifically the DEM raster X coordinate?

Reply:
Patched in 6adbea07c